### PR TITLE
Removes Enriched MSG Addiction

### PR DESCRIPTION
**Pull request by trazodont**
## About the PR 
Removes the addiction probability from Enriched MSG.

## Why's this needed? 
Discord discussion lead to many people voicing it made them uncomfortable from implications due to MSG's history.

## Changelog 

```changelog
(u)Shiiba
(+)Removes MSG addiction
```

### CODE REVIEW

**Comment by HatNJacket** [comment]
I think it would be a good idea to mention that you're also talking about enriched MSG so avoid confusion since both MSG and enriched MSG are in the game.

**Comment by RelentlessGarbage** [comment]
Hm. Not sure how I feel about this. I've always thought the implications of MSG/Enriched MSG being partially harmful were odd, but there is some balancing around enriched MSG as a chem (such as it showing up in food that's been "perfectly grilled"). I'm fine with the effects of regular MSG being altered/removed, but for Enriched MSG specifically, why not just change the chemical's name to something sci-fi and keep the old effects?

**Comment by trazodont** [comment]
> I think it would be a good idea to mention that you're also talking about enriched MSG so avoid confusion since both MSG and enriched MSG are in the game. Looking over it again it looks like you also removed the healing component of it which is a bit strange

Woops, removing the healing was a mistake, didn't catch that. I'll add it back later.

**Comment by ZeWaka** [comment]
^ based on some discussion

**Comment by M-Earthfire** [comment]
Enriched MSG would be, in my mind, the space science equivalent of MSG. And considering the over the top reaction of the consumption of 1u, i feel it is fitting to be highly addictive.

If it would be normal MSG, i would agree. But no enriched one.

**Comment by CaioFrancisco** [comment]
Ignoring the addiction discussion, which is something I don't want to opinate on, you also removed the overdose threshold for it on the PR. Was that accidental or intentional?

File: code/modules/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -3168,56 +3155,9 @@ datum
 				if (!volume_passed)
 					return ..()
 				if (method == INGEST)
-					var/datum/ailment/addiction/AD = M.addicted_to_reagent(src)
-					if (!AD)
-						boutput(M, "<B>THIS TASTES <font size='92'>~<font color='#FF0000'> A<font color='#FF9900'> M<font color='#FFff00'> A<font color='#00FF00'> Z<font color='#0000FF'> I<font color='#FF00FF'> N<font color='#660066'> G<font color='#000000'> ~ !</font></B>")
+					boutput(M, "<B>THIS TASTES <font size='92'>~<font color='#FF0000'> A<font color='#FF9900'> M<font color='#FFff00'> A<font color='#00FF00'> Z<font color='#0000FF'> I<font color='#FF00FF'> N<font color='#660066'> G<font color='#000000'> ~ !</font></B>")
 				..(M, method, volume_passed)
 
-			on_mob_life(var/mob/M, var/mult = 1)
```
Comment (Zonespace27) [inline code/modules/chemistry/Reagents-FoodDrink.dm:3176]: this removes the point of the *secret chem*, high healing while also being addictive

**Comment by Adharainspace** [comment]
I've removed the negative effects of MSG while leaving enriched MSG untouched for now. Closing this PR because of that.

**Pull request closed without merging**